internal/cmd/cmd.go

```diff
@@ -137,7 +137,7 @@ func (ctx *Context) Infof(format string, params ...interface{}) {
 		//level (since `Infof` calls `Logf` internally). This is done so
 		//that this function can produce more accurate source location
 		//debug information.
-		logger.Logf(context.TODO(), corelogger.INFO, format, params...)
+		logger.Logf(context.TODO(), corelogger.INFO, corelogger.Labels{}, format, params...)
```

Comment:
loggo has a LogWithLabelsf which our internal logger calls.
I wonder if we want to align our internal logger interface. We'd keep Logf without labels.

Reply:
I'm trying to keep the number of methods on the logger interface small, even at the cost of an additional argument. That way, when we potentially replace the underlying logger (one can dream!) the task is very much small and bounded.